tests/CMakeLists.txt

```diff
@@ -68,14 +68,11 @@ install (TARGETS qlacloserelays qlacommand eth1394Test instrument block1394eth e
          COMPONENT Amp1394-utils
          RUNTIME DESTINATION bin)
 
-if (ARM32 OR WIN32)
-  message (STATUS "Skipping dvrktest for ARM32 or WIN32 build")
+if (ARM32)
```

Comment:
This can be removed as well -- the reason I didn't compile on ARM32 was due to the experimental filesystem.